docs/HowToGuides/GettingStarted.md

```diff
@@ -259,10 +260,14 @@ Phew, that's a lot to digest! Now let's proceed to the actual build itself!
        --sccache --release-debuginfo --swift-disable-dead-stripping --test \
        --xcode
      ```
+   Linux (uses Ninja):
+     ```sh
+     utils/build-script --release-debuginfo --test --skip-early-swift-driver
+     ```
    This will create a directory
    `swift-project/build/Ninja-RelWithDebInfoAssert`
    (with `Xcode` instead of `Ninja` if you used `--xcode`)
-   containing the build artifacts.
+   containing the Swift compiler and standard library build artifacts.
```

Comment:
There are more build artifacts, including Clang and the LLVM tools. I think it would be better to not specify more details here.

Reply:
I think it's better to specify exactly what this first basic command builds for noobs, so I added clang/LLVM to the list instead. Let me know what you think.

Reply:
That's also reasonable.